extensions/common/context.cc

```diff
@@ -414,7 +414,7 @@ void populateExtendedHTTPRequestInfo(RequestInfo* request_info) {
   populateExtendedRequestInfo(request_info);
 
   getValue({"request", "referer"}, &request_info->referer);
-  getValue({"request", "user_agent"}, &request_info->user_agent);
+  getValue({"request", "useragent"}, &request_info->user_agent);
```

Comment:
@gargnupur @kyessenov is there some sort of linter we could write that would catch these issues at review time?

Reply:
Yeah something that will sync with source/extensions/filters/common/expr/context.h, to make sure we are using the right fields...

Reply:
Yeah, although here specifically it's using ABI, not CEL. ABI usage might be hard to statically verify.